### PR TITLE
Added my secret lesson file

### PR DESCRIPTION
**Pull request by nrei0**

### CODE REVIEW

**Comment by cptJackCrutch** [comment]
AAARRRGGGHH!!

File: search-highlight/index.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+window.addEventListener("DOMContentLoaded", () => {
+  // all DOM elements loaded.
+  const inputField = document.getElementById("input-field");
+  const contentEl = document.getElementById("content");
+
+  //   console.log(contentEl.innerText);
```
Comment (pl-a-st) [inline search-highlight/index.js]: WTF?

File: search-highlight/index.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      if (contentEl) {
+        contentEl.innerHTML = contentEl.innerText.replace(
+          new RegExp(value, "g"),
+          '<div style="background-color: red; display: inline;">' +
+            value +
+            "</div>"
+        );
+      }
+
+      //       if (contentEl) {
```
Comment (Lev1ossa) [inline search-highlight/index.js:35]: thx for hw help

File: search-highlight/index.js (hunk 1 above)
Comment (nrei0) [reply search-highlight/index.js]: I will remove

- [x] remove comments

**Pull request merged**